File: app/src/main/java/fr/insapp/insapp/models/UserSearchResults.java
package fr.insapp.insapp.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by thomas on 13/07/2017.
 */

public class UserSearchResults {

    @SerializedName("users")
    private List<User> users;

    public UserSearchResults(List<User> users) {
        this.users = users;
    }

    public List<User> getUsers() {
        return users;
    }
}

File: app/src/main/java/fr/insapp/insapp/models/UserSearchResults.kt
package fr.insapp.insapp.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by thomas on 13/07/2017.
 * Kotlin rewrite on 29/08/2019.
 */

@Parcelize
data class UserSearchResults(
    @SerializedName("users") val users: MutableList<User>
): Parcelable
